treeherder/perf/migrations/0010_fix_signature_uniqueness.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-09-04 09:38
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0008_remove_failure_match'),
```

Comment:
Could you adjust this for the new migrations on `master`? (`s/0008_remove_failure_match/0010_remove_runnable_job/`)

Reply:
> 
> 
> Could you adjust this for the new migrations on `master`? (`s/0008_remove_failure_match/0010_remove_runnable_job/`)

Done.

Reply:
Travis is failing; the dependency now references a deleted migration